partials/side-nav.hbs

```diff
@@ -30,48 +30,77 @@
 <nav class="side-bar hide">
     <button class="side-bar-close">{{> icons/close}}</button>
     <ul class="side-bar-list">
-        <li class="side-bar-title">Privacy Essentials</li>
+        <li class="side-bar-title">DuckDuckGo Search</li>
         <li>
           <a href="https://duckduckgo.com/">Private Search</a>
         </li>
         <li>
-          <a href="https://duckduckgo.com/app">App and Extension</a>
+          <a href="https://duckduckgo.com/settings#appearance">Themes</a>
+        </li>
+        <li>
+          <a href="https://duckduckgo.com/settings">All Settings</a>
+        </li>
+        <li>
+          <a href="https://duckduckgo.com/bangs">!Bang Search Shortcuts</a>
         </li>
 
-        <li class="side-bar-title">Why Privacy</li>
+        <li class="side-bar-title">Browser Downloads</li>
+        <li>
+          <a href="https://duckduckgo.com/ios">iOS</a>
+        </li>
         <li>
-          <a href="https://spreadprivacy.com/">Privacy Blog</a>
+          <a href="https://duckduckgo.com/android">Android</a>
         </li>
         <li>
-          <a href="https://duckduckgo.com/newsletter">Privacy Crash Course</a>
+          <a href="https://duckduckgo.com/mac">Mac</a>
+          <div style="display: inline-flex;justify-content: center;align-items: center;box-sizing: border-box;padding: 3px 5px;margin-left: 4px;background: #f9be1a;border-radius: 4px;font-weight: 600;font-size: 10px;line-height: 1;letter-spacing: 0.04px;color: rgba(0, 0, 0, 0.9);">NEW</div>
```

Comment:
Are the inline styles just for the demo? Can we update the code to use the class?